packages/vue-component-meta/src/index.ts

```diff
@@ -245,6 +250,18 @@ ${typeHelpersCode}
 			},
 		};
 
+		function getType() {
+
+			const $type = symbolProperties.find(prop => prop.escapedName === 'type');
+
+			if ($type) {
+				const type = typeChecker.getTypeOfSymbolAtLocation($type, symbolNode!);
+				return Number(typeChecker.typeToString(type));
+			}
+
+			return 0;
+		}
```

Comment:
great it was easier than I thought 😄  , so if the return is 0  => it is not a valid Vue component?

Reply:
Yes it's designed that way, I'm just not sure how applicable it is for your use case.

Reply:
very useful, as I'm  loading folders with components and *.stories.ts  so many are not exposing any vue component